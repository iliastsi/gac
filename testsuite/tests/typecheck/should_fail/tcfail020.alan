MulDimArray() : proc
  i : int[4];
{
  i = i;
}
