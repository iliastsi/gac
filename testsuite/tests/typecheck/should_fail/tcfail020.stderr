tcfail020.alan:4:3: Error: Type mismatch at `i = i;'
	Expected `int' or `byte' but expression `i' is of type `int [4]'

